### DISCUSSION TITLE
希望可以添加一个webSocket测试工具

### DISCUSSION THREAD

**Discussion by vince-0202** [he3-app/he3-feedback#79] [Ideas]
类似WebsocketMan 进行websocket连接测试

**Comment by hongchaodeng** [comment]
感谢你的反馈！已经创建了 issue 并在我们 roadmap 当中 😄

**Comment by lyzhang1999** [comment]
[https://t.he3app.com?tzta](https://t.he3app.com/?tzta)

**Reply by lyzhang1999** [reply]
您好，我们做了一个简易版本，后续工具会在 https://github.com/he3-app/tools-example 仓库开源。
